openai/api_resources/image.py

```diff
@@ -171,9 +190,9 @@ def _prepare_create_edit(
             api_version=api_version,
             organization=organization,
         )
-        _, api_version = cls._get_api_type_and_version(api_type, api_version)
+        api_type, api_version = cls._get_api_type_and_version(api_type, api_version)
 
-        url = cls._get_url("edits")
+        url = cls._get_url("edits", None, api_type=api_type, api_version=api_version)
```

Comment:
Unless the second parameter is a positional-only parameter (unlikely), I'd indicate the parameter name.

Reply:
fixed